src/sections/Kanvas/kanvas_banner.js

```diff
@@ -117,22 +119,20 @@ const BannerSectionWrapper = styled.div`
 
     @media screen and (max-width: 1700px) {
         .kanvas-logo {
-            width: 12rem;
-            /* right: 12rem; */
-            /* margin-top: -5rem; */
+        width: 20rem;
+        left: 5rem;
+        margin-top: -7rem;
```

Comment:
```suggestion
```

Reply:
```css
margin-top: -7rem;
```

is by default for the `.kanvas-logo`, unless we're modifying the value, writing it again is redundant.